src/build_loop.rs

```diff
@@ -57,8 +179,11 @@ impl<'a> BuildLoop<'a> {
     /// When new filesystem changes are detected while a build is
     /// still running, it is finished first before starting a new build.
     #[allow(clippy::drop_copy, clippy::zero_ptr)] // triggered by `select!`
-    pub fn forever(&mut self, tx: chan::Sender<Event>, rx_ping: chan::Receiver<()>) {
-        let send = |msg| tx.send(msg).expect("Failed to send an event");
+    pub fn forever(&mut self, tx: chan::Sender<LoopHandlerEvent>, rx_ping: chan::Receiver<()>) {
+        let send = |msg| {
+            debug!("BuildLoop sending"; "message" => format!("{:#?}", msg));
```

Comment:
This debug message seems likely to cause spam. It's probably useful for building this feature, but I suggest removing it before this PR is merged.